### PR TITLE
Pattern lab example

### PR DESCRIPTION
**Pull request by tekante**
Not sure what the trick is but I could never get localhost:3050 working. I tried defining both ports and expose options in the docker-compose.yml file and while docker ps would show a mapping (0.0.0.0:3050->3050/tcp) I'd never get a response trying to access it. The root of the dev machine (192.168.99.100:3050) would work as would the container specific IP address and the DNS named version. Not sure if the root of the issue is with docker or with the browser sync stuff that's listening on that port in the container :/ but my digging has not been illuminating thus far.

Adam here mostly in case he's interested. We should probably get Bloom and/or Evan to take a look though I wasn't getting their names to come up in the reviewers list for what I'm sure is some permission thing in github to track down.

### CODE REVIEW

**Comment by tekante** [comment]
Got the permission bits figured out so Evan and Bloom can see what seems to work for running all of pattern lab stuff from within the build containers.

File: pattern-lab/.devtools-watch-ignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.*___jb_tmp___
+.*___jb_old___
+\.*sw*
+
```
Comment (grayside) [inline pattern-lab/.devtools-watch-ignore]: Well, I didn't know this file was a thing. Seems like I should add this to p2-env to exclude `\build`

File: pattern-lab/install.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+##
+# Container for the initial install
+##
+version: '2.1'
+
+services:
```
Comment (grayside) [inline pattern-lab/install.yml]: Kind of a novel split here from cli.yml. Why not group them together?

Comment (tekante) [reply pattern-lab/install.yml]: They really could be combined (in which case I would return to the build.yml style naming). I was originally looking to break out things which run to completion automatically from things which should run interactively. That distinction and the proliferation of yaml files seems to fall a little flat now that you've prompted me to consider it more. I'll update to reconsolidate into a build.yml file.

**Pull request merged**